src/umpire/tpl/json/json.hpp

```diff
@@ -12610,6 +12610,14 @@ class serializer
         }
     }
 
+    template<typename NumberType, detail::enable_if_t<
```

Comment:
`std::enable_if_t` was added in C++14. Good, it looks like this is defined by the code on line 620 based on C++11 constructs.

Reply:
Is there something further I need to change here?  I'm not sure I follow the "Good" portion of the comment

Reply:
Sorry for the confusion. No, I was just commenting that it's nice that they defined `enable_if_t`!